content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/configuration.md

```diff
@@ -16,18 +16,57 @@ Since both O3DE Gems and projects use the same CMake build functions, these inst
 
 ## Build setup
 
-The first step of enabling the Multiplayer Gem for a project is to make sure that the `<ProjectName>.Static` target of your project includes the correct dependencies. Make the following edits to the `<ProjectName>.Static` target:
+Start by ensuring the Multiplayer Gem is [added to the project](/docs/user-guide/project-config/add-remove-gems/).
+After adding the Multiplayer Gem to the project there are more steps that require editing some of the project's CMake and C++ files by hand. 
+### CMakeList.txt changes
+Make sure the `<ProjectName>.Static` target includes the correct dependencies. 
+Find the CMake file defining your project's static target; for example `<ProjectName>/Code/CMakeList.txt`. 
```

Comment:
If I generate a new project with O3DE, there is a <ProjectName>/Gem/CmakeList.txt, theres is no <ProjectName/Code/CMakeList.txt. Should that be mentioned as an example here?

Reply:
Aye, that's why I chose "for example" since we have different project types. There are many locations where I suggest a path (using the default path for external projects; which I'm guessing you've create a project tied to the engine?)